src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -512,6 +512,18 @@ std::vector<MessagePtr> parseUserNoticeMessage(Channel *channel,
         }
     }
 
+    auto buildAndEmplaceMessage = [&builtMessages,
+                                   mirrored](MessageBuilder &builder) {
+        builder->flags.set(MessageFlag::Subscription);
+        if (mirrored)
+        {
+            builder->flags.set(MessageFlag::SharedMessage);
+        }
+
+        auto newMessage = builder.release();
+        builtMessages.emplace_back(newMessage);
+    };
```

Comment:
Are you wanting it to be duplicated so that the raid message isn't flagged as `Subscription`? I believe it should be as that's (currently) what dictates the background colour & highlighting.

If you'd rather it wasn't tagged as subscription, I'm more than happy to introduce some "event" (for lack of a better name) message type that has the same behaviour, but isn't tied to subscription bits (and isn't connected to subscription-related settings).

Otherwise, happy to revert this :+1:

Reply:
I think it's still best to revert so we don't end up thinking it's correct. If it still requires the sub flag, leave the sub flag but just in its own place

Reply:
Mint, will do 👍

Reply:
Re-inlined in [afe3487](https://github.com/Chatterino/chatterino2/pull/5651/commits/afe3487bd97d0b3d1be10050e07a4bd1c4d91df5)